### PR TITLE
chore: formatting

### PR DESCRIPTION
**Pull request by nobkd**
- formatting
- mostly whitespace changes
- some typo fixes

### CODE REVIEW

**Comment by tipiirai** [comment]
Wow. This is a major update. I tend to use extra new lines between things due to aesthetic reasons. Do you prefer a tighter formatting?

**Comment by nobkd** [comment]
I usually have only one newline to split ideas / functions / ..., this allows me to see more code in the short area of focus, while keeping a break between ideas / functions / ...

In e. g. Markdown lists I don't use extra lines, because they are separated through the `*`, `-` or `1.` indicator....

In the end, you can still decide if you like it or not. And I theoretically can also undo some changes.

File: packages/nuekit/src/cli.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ async function printVersion() {
 
 async function runCommand(args) {
   const { createKit } = await import('./nuekit.js')
-  const { cmd='serve', dryrun, deploy, root=null, port } = args
+  const { cmd='serve', dryrun, deploy, root = null, port } = args
```
Comment (nobkd) [inline packages/nuekit/src/cli.js]:
```suggestion
  const { cmd = 'serve', dryrun, deploy, root = null, port } = args
```

**Comment by nobkd** [comment]
Closing, and maybe later reopening with smaller scope.

**Pull request closed without merging**